fixtures/aputure/c300d.json

```diff
@@ -0,0 +1,263 @@
+          "type": "Effect",
+          "effectName": "Pulsing",
+          "switchChannels": {
+            "Undefined / Lightning Mode Options / Explosion Mode Options": "Undefined"
+          }
+        },
+        {
+          "dmxRange": [120, 139],
+          "type": "Effect",
+          "effectName": "Strobe",
+          "switchChannels": {
+            "Undefined / Lightning Mode Options / Explosion Mode Options": "Undefined"
+          }
+        },
+        {
+          "dmxRange": [140, 159],
+          "type": "Effect",
+          "effectName": "Explosion",
+          "switchChannels": {
+            "Undefined / Lightning Mode Options / Explosion Mode Options": "Explosion Mode Options"
+          }
+        },
+        {
+          "dmxRange": [160, 255],
+          "type": "NoFunction",
+          "switchChannels": {
+            "Undefined / Lightning Mode Options / Explosion Mode Options": "Undefined"
+          }
+        }
+      ]
+    },
+    "FX Frequency Control": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 19],
+          "comment": "1",
+          "helpWanted": "What speeds do these numbers represent?",
+          "type": "EffectSpeed",
+          "speedStart": "slow",
+          "speedEnd": "fast"
+        },
+        {
+          "dmxRange": [20, 39],
+          "comment": "2",
+          "type": "EffectSpeed",
+          "speedStart": "slow",
+          "speedEnd": "fast"
+        },
+        {
+          "dmxRange": [40, 59],
+          "comment": "3",
+          "type": "EffectSpeed",
+          "speedStart": "slow",
+          "speedEnd": "fast"
+        },
+        {
+          "dmxRange": [60, 79],
+          "comment": "4",
+          "type": "EffectSpeed",
+          "speedStart": "slow",
+          "speedEnd": "fast"
+        },
+        {
+          "dmxRange": [80, 99],
+          "comment": "5",
+          "type": "EffectSpeed",
+          "speedStart": "slow",
+          "speedEnd": "fast"
+        },
+        {
+          "dmxRange": [100, 119],
+          "comment": "6",
+          "type": "EffectSpeed",
+          "speedStart": "slow",
+          "speedEnd": "fast"
+        },
+        {
+          "dmxRange": [120, 139],
+          "comment": "7",
+          "type": "EffectSpeed",
+          "speedStart": "slow",
+          "speedEnd": "fast"
+        },
+        {
+          "dmxRange": [140, 159],
+          "comment": "8",
+          "type": "EffectSpeed",
+          "speedStart": "slow",
+          "speedEnd": "fast"
+        },
+        {
+          "dmxRange": [160, 179],
+          "comment": "9",
+          "type": "EffectSpeed",
+          "speedStart": "slow",
+          "speedEnd": "fast"
+        },
+        {
+          "dmxRange": [180, 199],
+          "comment": "10",
+          "type": "EffectSpeed",
+          "speedStart": "slow",
+          "speedEnd": "fast"
+        },
+        {
+          "dmxRange": [200, 219],
+          "comment": "R",
```

Comment:
I would have omitted the comments if they were just numbers, but I added them incase "R" has some special meaning I'm not aware of.

Reply:
Maybe it's "reverse"? But I agree, let's leave the comments in.